src/components/QuicksightDashboardPage.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import { Store } from "redux";
+import { RootState } from "../store";
+import * as PropTypes from "prop-types";
+import Header from "./Header";
+import Footer from "./Footer";
+import title from "../utils/title";
+import QuicksightDashboard from "./QuicksightDashboard";
+
+export interface QuicksightDashboardPageProps extends React.Props<QuicksightDashboardPageProps> {
+  params: {
+    library?: string;
+  };
+}
+
+export interface QuicksightDashboardPageContext {
+  editorStore: Store<RootState>;
+}
+
+/** Page holds quicksight dashboards. */
+export default class DashboardPage extends React.Component<QuicksightDashboardPageProps> {
+  context: QuicksightDashboardPageContext;
+
+  static contextTypes: React.ValidationMap<QuicksightDashboardPageContext> = {
+    editorStore: PropTypes.object.isRequired as React.Validator<Store>,
+  };
+
+  static childContextTypes: React.ValidationMap<object> = {
+    library: PropTypes.func,
+  };
+
+
+  getChildContext() {
+    return {
+      library: () => this.props.params.library,
+    };
+  }
+
+
+  render(): JSX.Element {
+    const { library } = this.props.params;
+    return (
+      <div className="quicksight-dashboard">
+        <Header />
+        <main className="body">
+          <QuicksightDashboard dashboardId="library"/>
```

Comment:
It's not. The dashboard is the library / librarian dashboard view.

Reply:
Since it's a hardcoded value, I'm not sure why we need to pass it as a `prop`. Can we just remove the `prop` here and use the hardcoded value inside the child `QuicksightDashboard` component?

Reply:
It doesn't look like this has been updated.